src/Core/Services/ApiService.cs

```diff
@@ -767,6 +767,12 @@ public async Task<string> GetUsernameFromAsync(ForwardedEmailServiceType service
                         break;
                     case ForwardedEmailServiceType.FirefoxRelay:
                         requestMessage.Headers.Add("Authorization", $"Token {config.ApiToken}");
+                        requestMessage.Content = new StringContent(JsonConvert.SerializeObject(
```

Comment:
Nice changes, I just think that Instead of `JObject` and `JProperty` we could use an anonymous object making the code cleaner

`requestMessage.Content = new StringContent(JsonConvert.SerializeObject(
                            new { enabled = true, description = "Generated by Bitwarden."}), Encoding.UTF8, "application/json");`

Reply:
Very neat idea!